conda_smithy/linters_forge_yml.py

```diff
@@ -0,0 +1,87 @@
+import json
+from inspect import cleandoc
+from textwrap import indent
+from typing import List
+
+import jsonschema
+from pydantic import BaseModel
```

Comment:
I would like and strongly propose to keep `pydantic` here. Pydantic provides an elegant way to find additional fields that are not part of the schema (see my implementation) - and also, I think the pydantic model should be used in more parts of the smithy code anyway since it provides a type-safe way to access fields.

Reply:
I strongly disagree. This PR use pydantic to check for which fields to hint about. If we can't do that using the json schema we are essentially making a distinction between pydantic model and json schema.

Reply:
What do you mean with "distinction"? The pydantic model should be equivalent to the JSON schema anyway - if it's not, we have a bug. I am not aware of a good way of doing this with JSON schema.